src/sentry/static/sentry/app/components/list/symbol/bullet.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+type Props = {
+  className?: string;
+};
+
+const Bullet = styled(({className}: Props) => <div className={className} />)`
```

Comment:
I understand needing to use additional DOM to render the coloured numbers. For simple markers like bullets and numbers CSS can do that pretty well, and using additional DOM feels like we're making something that should be simple into a more complex construct. Is it hard to only use additional DOM when CSS can't do what we want?

Reply:
I agree...I will update the component 😉 Thanks for the feedback.